content/cloud/pricing-and-billing.md

```diff
@@ -0,0 +1,114 @@
+title = "Pricing Plans and Billing"
+template = "cloud_main"
+date = "2023-06-20T00:07:00Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/payment-and=billing.md"
+
+---
+- [Fermyon Cloud Pricing](#fermyon-cloud-pricing)
+- [General Information](#general-information)
+  - [Payment](#payment)
+  - [Refund](#refund)
+  - [Support](#support)
+  - [Quota](#quota) 
+- [Next Steps](#next-steps)
+
+## Fermyon Cloud Pricing
+Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quota](#quota).
+
+As described in detail on the [Fermyon Pricing page](fermyon.com/pricing), there are 3 different plans crafted to fit your workloads' requirements: Starter, Growth, and Enterprise. 
+
+The Starter Plan provides a broad range of Fermyon Cloud services for you to try out, free of charger, forever. If you find that you're hitting the upper limits of the Starer Plan, we recommend promoting your account to the Growth Plan, where you can enjoy the same services with a higher quota limit. Lastly, if the serives offered don't quite meet your needs in the Growth Plan, [reach out](mailto:sales@fermyon.com) to the Fermyon team and we'll work to create a customized Enterprise Plan that will best meet your requirements. 
+
+> Please note that Growth and Enterprise Plans are billed on a monthly reoccuring basis. Start Plan is a free subscription, without an expiration date. 
+
+## General Information
+
+### Payment
+
+**I just signed up for the Growth Plan, when should I expect my first charge?**
+
+You should receive your first charge immediately after signing up for the Growth Plan. You can expect your second charge on the monthly anniversary of your signup.
+
+**What payment methods for you accept?**
+
+Most. One of the main reasons we chose to work with industry leader Stripe is because of the huge scope of payment options they offer, starting with the normal credit / debit card options, but extending to ACH, SEPA, Apple Pay / Google Pay and many more including a larger variety of global options. For more details and a list to all Stripe payment options see this Stripe Resource. 
```

Comment:
Spare me the Stripe sales pitch #badattitude - list the important ones (maybe bulleted), link for other ones, and don't clutter it telling me how awesome Stripe is.

```suggestion
We accept all forms of payment supported by our payment partner Stripe. These include:

* Credit and debit cards
* ACH
* SEPA
* Apple Pay / Google Pay

For a full list to accepted payment methods see ***STRIPE LINK*** 
```